### PR TITLE
Clean issues

### LINKED ISSUE #1560: Disable lock/stake calls.

### LINKED ISSUE #1604: KtonStaking is not distributing rewards after latest runtime upgrade on Crab
https://crab-scan.darwinia.network/address/0x652182C6aBc0bBE41b5702b05a26d109A405EAcA?tab=txs

### PR DESCRIPTION
**Pull request by AurevoirXavier**
```
subalfred storage-key --pallet BridgeKusamaGrandpa --item ImportedHeaders
0xad61a448d2290042116be82bf61332ebf7e5937ed7be0ea26416b155b0109b48
```

- Remove Kusama header migration code
- Typo fixes
- Closes #1604
- Closes #1560
- Closes https://github.com/darwinia-network/KtonDAO/issues/23

image

### CODE REVIEW

File: runtime/crab/src/migration.rs [hunk 1]
```diff
@@ -45,47 +45,5 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 }
 
 fn migrate() -> frame_support::weights::Weight {
-	// dawinia
-	use darwinia_staking::CacheState;
-	if let Some(s) = migration::get_storage_value::<(CacheState, CacheState, CacheState)>(
-		b"DarwinaStaking",
```
Comment (hackfisher) [inline runtime/crab/src/migration.rs:51]: It seems that here is also a typo, does it mean this migration was not successfully executed?

What's the impact for Crab.

**Comment by hackfisher** [comment]
Please also complete step 2 in this issue to close it: https://github.com/darwinia-network/KtonDAO/issues/23

File: runtime/crab/src/migration.rs (hunk 1 above)
Comment (AurevoirXavier) [reply runtime/crab/src/migration.rs:51]: Get `None`, then skip the `put`.

Fallback to use the https://github.com/darwinia-network/darwinia/blob/1ac5283f5baaf351d472de87bc162d373ee4897e/pallet/staking/src/lib.rs#L244. This only affects the first session. The election might also be skipped.

Comment (hackfisher) [reply runtime/crab/src/migration.rs:51]: If `ExposureCacheStates ` is renamed and not used anymore, should we clean its storage?

Comment (AurevoirXavier) [reply runtime/crab/src/migration.rs:51]: There is no need for now. There are many unnecessary items. Clean them all up in Staking V3 at once.

**Pull request merged**